release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -143,6 +143,13 @@ In earlier versions of {product-title}, Ironic container images used {op-system-
 
 For more information about the Ironic provisioning service, see xref:../installing/installing_bare_metal_ipi/ipi-install-overview.adoc[Deploying installer-provisioned clusters on bare metal].
 
+[id="ocp-4-12-openstack-external-cloud-control-upgrade"]
+=== Cloud provider configuration updates for clusters that run on {rh-openstack}
+
+In {product-title} 4.12, clusters that run on {rh-openstack-first} are switched from the legacy {rh-openstack-first} cloud provider to the external Cloud Controller Manager (CCM). This change follows the move in Kubernetes from in-tree, legacy cloud providers to external cloud providers that are implemented by using the link:https://kubernetes.io/docs/concepts/architecture/cloud-controller/[Cloud Controller Manager].
```

Comment:
I'm wondering if we should use 'OpenStack' instead of 'Red Hat OpenStack Platform (RHOSP)' when refering to the cloud provider, as it's not something provided/developed by Red Hat.
@MaysaMacedo please correct me if I'm wrong

```suggestion
In {product-title} 4.12, clusters that run on {rh-openstack-first} are switched from the legacy OpenStack cloud provider to the external Cloud Controller Manager (CCM). This change follows the move in Kubernetes from in-tree, legacy cloud providers to external cloud providers that are implemented by using the link:https://kubernetes.io/docs/concepts/architecture/cloud-controller/[Cloud Controller Manager].
```

Reply:
We have [openshift/cloud-provider-openstack](https://github.com/openshift/cloud-provider-openstack) now. Probably just OpenStack works better.